src/views/components/trace/trace-search.vue

```diff
@@ -95,12 +95,19 @@
     @Prop({default: {label: 'All', key: ''}})
     private service!: Option;
     private instance: Option = {label: 'All', key: ''};
-    private endpointName: string = localStorage.getItem('endpointName') || '';
-    private traceId: string = localStorage.getItem('traceId') || '';
+    private endpointName: string = this.getUrlKey('endpointname') || localStorage.getItem('endpointName') || '';
+    private traceId: string = this.getUrlKey('traceid') || localStorage.getItem('traceId') || '';
     private traceState: Option = {label: 'All', key: 'ALL'};
     @Prop({default: false, type: Boolean})
     private inTopo!: boolean;
 
+    private getUrlKey(name : any) {
```

Comment:
It doesn't need to be that complicated, please use "this.$route.query". ref: https://router.vuejs.org/guide/essentials/dynamic-matching.html#dynamic-route-matching

Reply:
@x22x22 Thanks for your review! Based on your comments, I have updated the codes, pls continue to review and merge.